src/main/java/com/ning/billing/recurly/model/ShippingMethod.java

```diff
@@ -58,15 +58,6 @@ public class ShippingMethod extends RecurlyObject {
     @XmlElement(name = "updated_at")
     private DateTime updatedAt;
 
-    @XmlElement(name = "liability_gl_account_id")
-    private String liabilityGlAccountId;
-
-    @XmlElement(name = "revenue_gl_account_id")
-    private String revenueGlAccountId;
-
-    @XmlElement(name = "performance_obligation_id")
-    private String performanceObligationId;
-
```

Comment:
These were somehow added in twice, so this is removing the 2nd instance of these attrs. Check up from lines 46-52 to see these are still in the ShippingMethod model.